### DISCUSSION TITLE
Support for WIN11 IoT Enterprise LTSC 2024 - 24H2 - 26100.1.240331

### DISCUSSION THREAD

**Discussion by sailsoft-hu** [valinet/ExplorerPatcher#3369] [Q&A]
Hi,
Are you planning to support the EP for the 24H2 - 26100.1.240331 build?
The latest version (22621.3527.65.5), it can be installed after renaming ep_setup.exe, but the functions are only available to a limited extent.
Thank you in advance for your answer

**Comment by 0x8008** [comment]
Basing on my own experience, everything that isn't related to restoring the Win10 taskbar works just fine.

**Reply by sailsoft-hu** [reply]
For me, the WIN10 taskbar would be the most important. :(

**Comment by pyrates999** [comment]
The windows 10 taskbar code as far as I know has been removed in windows 11 24h2.

**Comment by idbssoft** [comment]
How come StartAllback has a taskbar of W10 for 24H2? Only I miss the W10 StartMenu.

![SS26100863-](https://github.com/valinet/ExplorerPatcher/assets/48468972/1577b2b7-b1dc-463c-b27f-738539b63374)

**Reply by pyrates999** [reply]
You can ask that in the startallback forums.
